### PR TITLE
[v7r2] hackathon fixes

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*RSS
FIX: VO field fixes

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Agent/JobAgent.py [hunk 1]
```diff
@@ -535,7 +535,7 @@ def _submitJob(self, jobID, jobParams, resourceParams, optimizerParams,
 
     wrapperFile = result['Value']
     jobReport.setJobStatus(status='Matched',
-                           minor='Could Not Extract JDL Parameters',
+                           minor='Submitted To CE',
```
Comment (atsareg) [inline WorkloadManagementSystem/Agent/JobAgent.py:538]: The wrapper is not yet submitted at this point

```suggestion
                           minor='Submitting To CE',
```

Comment (fstagni) [reply WorkloadManagementSystem/Agent/JobAgent.py]: Done.

File: ProductionSystem/Service/ProductionManagerHandler.py [hunk 2]
```diff
@@ -17,11 +17,15 @@
...
 
-  def setDatabase(self, oDatabase):
-    global database
-    database = oDatabase
+  @classmethod
+  def initializeHandler(cls, serviceInfoDict):
+    """ Initialization of DB object
+    """
+
+    cls.gProductionDB = ProductionDB()
```
Comment (atsareg) [inline ProductionSystem/Service/ProductionManagerHandler.py:27]: I do not think that we should prepend "g" to the name of the database here. It used to denote global objects. Now this is a class object which is in the code clearly seen as always used as cls.productionDB

File: ProductionSystem/Service/ProductionManagerHandler.py [hunk 3]
```diff
@@ -17,11 +17,15 @@
 transTypes = [six.string_types, int, list]
 
 
-class ProductionManagerHandlerBase(RequestHandler):
```
Comment (atsareg) [inline ProductionSystem/Service/ProductionManagerHandler.py:20]: Are you sure about this change ? I guess the idea was that specific ProductionManagers in extensions can start from this base class.

Comment (fstagni) [reply ProductionSystem/Service/ProductionManagerHandler.py:20]: Well, TransformationManager is an example of an extended (by LHCb) database without the need for this trick.

Comment (atsareg) [reply ProductionSystem/Service/ProductionManagerHandler.py:20]: This is fine with me. I just always try to understand the developer reasoning behind this or that solution.

Comment (arrabito) [reply ProductionSystem/Service/ProductionManagerHandler.py:20]: Honestly I don't remember. If you think that there is no need for this it's also fine with me.

File: ProductionSystem/Service/ProductionManagerHandler.py (hunk 2 above)
Comment (fstagni) [reply ProductionSystem/Service/ProductionManagerHandler.py]: I changed all.

**Review by atsareg — APPROVED** [review]

**Pull request merged**